lang/sk.json

```diff
@@ -145,141 +145,143 @@
 			"207": "Prekopírovaný súbor bol premenovaný na \"{name}\".",
 			"300": "Presunutie súborov zlyhalo.",
 			"301": "Kopírovanie súborov zlyhalo.",
-			"302": "[MISSING]Deleting file(s) failed.",
+			"302": "Mazanie súborov zlyhalo.",
 			"500": "Prehliadanie súborov je zakázané kvôli bezpečnosti. Kontaktujte prosím administrátora a overte nastavenia v konfiguračnom súbore pre CKFinder.",
 			"501": "Momentálne nie je zapnutá podpora pre generáciu miniobrázkov."
 		},
 		"fileInvalidCharacters": "Súbor nesmie obsahovať žiadny z nasledujúcich znakov: {disallowedCharacters}",
-		"fileNameNotEmpty": "[MISSING]The file name cannot be empty.",
+		"fileNameNotEmpty": "Názov súboru nesmie byť bez znakov.",
 		"folderInvalidCharacters": "Adresár nesmie obsahovať žiadny z nasledujúcich znakov: {disallowedCharacters}",
 		"incorrectExtension": "V tomto adresári nie je povolený tento formát súboru.",
-		"missingFile": "[MISSING]The requested file is no longer available.",
-		"missingFolder": "[MISSING]The folder you are trying to reach is no longer available.",
+		"missingFile": "Súbor už nie je k dispozícii.",
+		"missingFolder": "Priečinok už nie je k dispozícii.",
 		"noUploadFolderSelected": "Pred náhrávaním zvoľte adresár, prosím",
-		"operationCompleted": "[MISSING]Operation completed with errors.",
+		"operationCompleted": "Úloha dokončená s chybami.",
 		"unknown": "Server nemohol dokončiť spracovanie požiadavky. (Chyba {number})",
 		"unknownUploadError": "V priebehu posielania súboru sa vyskytla chyba.",
-		"uploadErrors": "[MISSING]Upload finished with errors.",
+		"uploadErrors": "Nahrávanie ukončené s chybami.",
 		"uploadPermissions": "Nahratie súboru nie je povolené."
 	},
 	"chooseResizedImage": {
-		"title": "[MISSING]Choose Resized",
-		"originalSize": "[MISSING]Original Size",
+		"title": "Vybrať upravenú verziu obrázku",
```

Comment:
Any chance to make it shorter? This is a toolbar button label, so it should be as compact as possible :(
![image](https://cloud.githubusercontent.com/assets/538321/14279524/ef7cf3d0-fb2d-11e5-8d42-81e602d5f5cc.png)